src/components/sidebar/Sidebar.scss

```diff
@@ -1,26 +1,26 @@
-/* #region menu */
```

Comment:
just a small heads up: you can keep this kind of comment as it makes easy navigating throughout the styles.